ydb/library/yql/public/udf/arrow/udf_arrow_helpers.h

```diff
@@ -123,6 +124,12 @@ class TSimpleArrowUdfImpl : public TBoxedValue {
             for (ui32 i = 0; i < ArgArrowTypes_.size(); ++i) {
                 bool isScalar;
                 ui64 length;
+                // XXX: If no value is given to the UDF, pass the Null datum as
+                // an optional argument value to the fixed-arg kernel.
+                if (!args[i].HasValue()) {
```

Comment:
if !args[i]. HasValue has different sematic for nested Optionals

Reply:
Thanks! [Fixed](https://github.com/ydb-platform/ydb/pull/3100/files#diff-0b96ada35f11c5251304bd4786016a2fe25410a591d3c904cba44911909bb967R131), left the comment.